src/main/java/net/sf/jabref/gui/EntryTypeDialog.java

```diff
@@ -91,12 +112,17 @@ private JPanel createEntryGroupsPanel() {
         JPanel panel = new JPanel();
         panel.setLayout(new VerticalLayout());
 
-        if(biblatexMode) {
+        if (biblatexMode) {
             panel.add(createEntryGroupPanel("BibLateX", EntryTypes.getAllValues(bibDatabaseContext.getMode())));
         } else {
             panel.add(createEntryGroupPanel("BibTeX", BibtexEntryTypes.ALL));
             panel.add(createEntryGroupPanel("IEEETran", IEEETranEntryTypes.ALL));
-            panel.add(createEntryGroupPanel(Localization.lang("Custom"), CustomEntryTypesManager.ALL));
+
+            if (!CustomEntryTypesManager.ALL.isEmpty()) {
```

Comment:
The custom entry types and the id based entry are not displayed in BibLatexMode. Is this by design? @JabRef/developers